src/workerd/server/tests/python/durable-object/worker.py

```diff
@@ -20,6 +24,12 @@ async def no_args_method(self):
     async def args_method(self, arg):
         return "value from python " + arg
 
+    def jspi_method(self, arg):
```

Comment:
Async def

Reply:
It's intentionally not async since it's testing JSPI. It should be supported to define synchronous methods on DO's right?

Reply:
Nope
All DO exported functions used over jsrpc should be async

Reply:
It probably works because we have some async wrapper around it